src/content/mod_configuration.cpp

```diff
@@ -270,8 +301,18 @@ void ModConfiguration::resolveDependencies()
 				++it;
 			}
 		}
+		if (last_mod_spec.has_value())
+			last_mod_spec->unsatisfied_depends.erase(mod.name);
 	}
 
-	// Step 4: write back list of unsatisfied mods
+	// Write back list of unsatisfied mods
 	m_unsatisfied_mods.assign(unsatisfied.begin(), unsatisfied.end());
+
+	// Check and add last mod
+	if (last_mod_spec.has_value()) {
+		if (!last_mod_spec->unsatisfied_depends.empty())
+			throw ModError("Mod selected as last by the game has an unsatisfied dependency.");
```

Comment:
can't this just be written to `m_unsatisfied_mods` and left to normal error handling?

Reply:
that should work